### ISSUE TITLE
2.8版本加载预训练模型不成功

### ISSUE THREAD

**Issue by sdreamforchen** [PaddlePaddle/PaddleDetection#9258]
### 问题确认 Search before asking

- [X] 我已经查询[历史issue](https://github.com/PaddlePaddle/PaddleDetection/issues)，没有发现相似的bug。I have searched the [issues](https://github.com/PaddlePaddle/PaddleDetection/issues) and found no similar bug report.

### Bug组件 Bug Component

Training

### Bug描述 Describe the Bug

1 使用cfg配置pretrain_weights: /workspace/Paddle/PaddleDetection1/output/softmax.pdparams加载不成功
2 在训练指令中使用 -o weights=pretrain_weights: /workspace/Paddle/PaddleDetection1/output/softmax.pdparams加载也不成功
而且没有任何报错

### 复现环境 Environment

paddledetection2.8

### Bug描述确认 Bug description confirmation

- [X] 我确认已经提供了Bug复现步骤、代码改动说明、以及环境信息，确认问题是可以复现的。I confirm that the bug replication steps, code change instructions, and environment information have been provided, and the problem can be reproduced.

### 是否愿意提交PR？ Are you willing to submit a PR?

- [X] 我愿意提交PR！I'd like to help by submitting a PR!
